### PR TITLE
chore: release v8.19.0

### PR DESCRIPTION
**Pull request by mejo-**
## [v8.19.0](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.19.0) (2024-09-16)
[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v8.18.0...v8.19.0)

### 🚀 Enhancements
* feat(NcModal): make dark header configurable [\#6083](https://github.com/nextcloud-libraries/nextcloud-vue/pull/6083) \([mejo-](https://github.com/mejo-)\)
* feat(NcActions): Emit `closed` event only when the actions are fully closed [\#6065](https://github.com/nextcloud-libraries/nextcloud-vue/pull/6065) \([susnux](https://github.com/susnux)\)

### 🐛 Fixed bugs
* fix(NcAppNavigationItem): align utils with actions and other components [\#6054](https://github.com/nextcloud-libraries/nextcloud-vue/pull/6054) \([GVodyanov](https://github.com/GVodyanov)\)
* fix(NcAvatar): add a fallback for missing CSS variable [\#6090](https://github.com/nextcloud-libraries/nextcloud-vue/pull/6090) \([Antreesy](https://github.com/Antreesy)\)

### Changed
* Updated workflows and dependencies

### CODE REVIEW

**Comment by susnux** [comment]
> fix(NcModal): make dark header configurable https://github.com/nextcloud-libraries/nextcloud-vue/pull/6083 ([@mejo-](https://github.com/mejo-))

Would consider this a feature

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nextcloud/vue",
-  "version": "8.18.0",
+  "version": "8.18.1",
```
Comment (susnux) [inline package.json:3]:
```suggestion
  "version": "8.19.0",
```

Semantically new props are feature

**Comment by susnux** [comment]
Please can we include also this:
- [ ] https://github.com/nextcloud-libraries/nextcloud-vue/pull/6065

File: package.json (hunk 1 above)
Comment (mejo-) [reply package.json:3]: Alright :blush:

**Comment by mejo-** [comment]
@susnux everything included now :blush:

**Comment by Antreesy** [comment]
Sorry, 
- #6097 is on the way
I can rebase later when it's in

**Review by Antreesy — APPROVED** [review]

**Review by ShGKme — APPROVED** [review]

**Pull request merged**